WordPress/Classes/Extensions/UIImageView+Blavatar.swift

```diff
@@ -0,0 +1,159 @@
+import Foundation
+
+
+/// UIImageView Helper Methods that allow us to download a Blavatar, given a website's "Icon Path"
```

Comment:
I think that unless you're actually doing something specific to blavatar (e.g. `isBlavatarURL`), I'd drop the "Blavatar" terminology from the code and use `SiteIcon` everywhere